Source/WebKit/UIProcess/API/APIDataTask.cpp

```diff
@@ -35,6 +35,11 @@ namespace API {
 
 DataTask::~DataTask() = default;
 
+Ref<DataTaskClient> DataTask::protectedClient() const
```

Comment:
nit: Perhaps this function should not be const because the caller can use the return value to modify the m_client member variable. Same for the other protected getter's being added here.

Reply:
This is the convention we use. Accessors to a member variable stored in Ref/RefPtr is `const` even if it returns mutable variant.